Index: .classpath
===================================================================
--- .classpath	(revision 0)
+++ .classpath	(revision 0)
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" output="build/classes" path="src"/>
+	<classpathentry kind="src" output="build/tck/classes" path="tck"/>
+	<classpathentry kind="lib" path="build/dist/javax.inject.jar"/>
+	<classpathentry kind="lib" path="build/tck/dist/javax.inject-tck.jar"/>
+	<classpathentry kind="lib" path="lib/junit-src.jar"/>
+	<classpathentry kind="lib" path="lib/junit.jar" sourcepath="/Users/pmuir/.m2/repository/junit/junit/3.8.1/junit-3.8.1-sources.jar"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="output" path="build/classes"/>
+</classpath>
Index: .project
===================================================================
--- .project	(revision 0)
+++ .project	(revision 0)
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>atinject</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+	</natures>
+</projectDescription>
Index: build.sh
===================================================================
--- build.sh	(revision 46)
+++ build.sh	(working copy)
@@ -9,8 +9,8 @@
 mkdir -p build/tck/dist
 
 # Compile classes.
-javac -g -d build/classes `find src -name *.java`
-javac -g -classpath build/classes:lib/junit.jar -d build/tck/classes \
+javac -source 1.5 -target 1.5 -g -d build/classes `find src -name *.java`
+javac -source 1.5 -target 1.5 -g -classpath build/classes:lib/junit.jar -d build/tck/classes \
 	`find tck -name *.java`
 
 FOOTER="<font size='-1'>Copyright (C) 2009 <a href='http://code.google.com/p/atinject/'>\
Index: tck-pom.xml
===================================================================
--- tck-pom.xml	(revision 0)
+++ tck-pom.xml	(revision 0)
@@ -0,0 +1,21 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.atinject</groupId>
+  <artifactId>inject-tck</artifactId>
+  <version>1.0.0-PFD-3</version>
+
+  <packaging>jar</packaging>
+
+  <name>JSR-330 TCK: Dependency Injection for Java</name>
+  
+  <url>http://code.google.com/p/atinject/</url>
+
+  <build>
+
+  </build>
+
+</project>
+
Index: pom.xml
===================================================================
--- pom.xml	(revision 46)
+++ pom.xml	(working copy)
@@ -5,7 +5,7 @@
 
     <groupId>javax.inject</groupId>
     <artifactId>javax.inject</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-PFD-1</version>
 
     <name>Javax Inject from the JSR-330 Expert Group</name>
 
Index: tck-readme.txt
===================================================================
--- tck-readme.txt	(revision 0)
+++ tck-readme.txt	(revision 0)
@@ -0,0 +1 @@
+mvn deploy:deploy-file -DpomFile=tck-pom.xml -Dfile=build/tck/dist/javax.inject-tck.jar -Durl=file:///Users/pmuir/development/repository && mvn deploy:deploy-file -DpomFile=tck-pom.xml -Dfile=build/tck/dist/javax.inject-tck-src.zip -Durl=file:///Users/pmuir/development/repository -Dclassifier=sources && mvn deploy:deploy-file -DpomFile=tck-pom.xml -Dfile=build/tck/dist/javax.inject-tck-javadoc.zip -Durl=file:///Users/pmuir/development/repository -Dclassifier=javadoc
